### ISSUE TITLE
Create hidden config directory under Linux

### ISSUE THREAD

**Issue by ghost** [TrsstProject/trsst#5]
On starting the client on Linux, it creates two directories ("Trsst Keyfiles" and "Trsst System Cache") in the user's home directory. 

It would be great if these directories could - conforming to usual Linux standards - be created as subdirectories of a hidden directory, e.g. ".trsst".

**Comment by vinipsmaker** [comment]
Actually, the standard on GNU/Linux is to use:
- .config/trsst for textual config files.
- .local/share/trsst for any non-textual config file (such as binary data, themes and config).
- .cache/trsst for cache files that can be safely erased without causing further problems.

Just search for freedesktop.org standards and you'll see.

**Comment by mmpowers** [comment]
Right: we'll need to create platform-specific behavior like we're already doing for OSX.  

In the meantime, a workaround is to set the "com.trsst.client.storage" and "com.trsst.server.storage" runtime properties at the command line, like so: 

java -Dcom.trsst.client.storage="~/.trsstd" -jar target/trsst-client-0.2-SNAPSHOT-exe.jar

**Comment by mmpowers** [comment]
By way of background, we used to use trsstd for everything, but separated out the 'Keyfiles' as things that you do _not_ want to delete, like your keystore and original copies of your feed and entries.  You usually don't want to delete the 'System Cache' either, because it's an off-line copy of all the feeds you're following as well as full-text search indexes of each, but you could delete it and still function.  So I'm not sure if "Cache" is the right word for that.

**Comment by vinipsmaker** [comment]
> You usually don't want to delete the 'System Cache' either, because it's an off-line copy of all the feeds you're following as well as full-text search indexes of each, but you could delete it and still function. So I'm not sure if "Cache" is the right word for that.

You can delete with no major harm, then it goes into the cache folder, even if the term is not accurate.

Imagine the cache folder as a way to free space used by tools like that _CCleaner_ from _Windows_. Or also if the user wants to move config file from one machine to another. It'll transfer less files to his low-speed pen drive if he only copy the `.config` and `.local` folders.

And users don't delete the contents of this folder frequently, then it shouldn't increase the server load significantly.

And increasing background info, it's not uncommon for applications to merge `.local/share` and `.config` into a folder inside `.config` and completely ignore `.local/share`.
